src/pages/Home.jsx

```diff
@@ -148,45 +154,30 @@ const Home = () => {
     },
   ];
 
-  const loadListings = async () => {
-
+  const handleFilterChanged = async (params) => {
     setLoading(true)
-    const auctionListings = await getAuctionListings(
-      contracts.MarketContract,
-      contracts.ZoombiesContract,
-      filters,
-      sortBy
-    );
-    setListings(auctionListings.map((listing) => ({
-      ...listing,
-      id: listing._id
-    })));
-    setLoading(false)
-  };
-
-
-
-  const handleRowClick = ({row}) => {
-    history.push(`/listing/${row.itemNumber}`);
-  };
-
-  const handleFilterChanged = (params) => {
     setFilters({ ...filters, ...params })
+    remove()
   }
 
   const handleSortByChanged = (attribute) => {
-    setSortBy({ ...sortBy, ...attribute })
+    // setSortBy({ ...sortBy, ...attribute })
+    setLoading(true)
+    setFilters({ ...filters, ...{
+      sortField: attribute.field
+    }})
+    remove()
   }
 
   useEffect(() => {
```

Comment:
Does this useEffect do anything then? If not can you remove it?

Reply:
yes. now it does.

but i will simplify it